plugins/Jobs.cpp

```diff
@@ -19,8 +19,14 @@ void BedrockPlugin_Jobs::upgradeDatabase(SQLite& db) {
                                    "repeat   TEXT NOT NULL, "
                                    "data     TEXT NOT NULL, "
                                    "priority INTEGER NOT NULL DEFAULT " + SToStr(JOBS_DEFAULT_PRIORITY) + ", "
-                                   "parentJobID INTEGER NOT NULL DEFAULT 0 )",
-                           ignore));
+                                   "parentJobID INTEGER NOT NULL DEFAULT 0, "
+                                   "timeout  TEXT DEFAULT NULL)",
```

Comment:
Every other column is `NOT NULL`; please follow suit and just use an empty string to signify that lack of timeout.